std/java/Lib.hx

```diff
@@ -30,6 +30,20 @@ package java;
 {
 
 	/**
+		Print the specified value on the default output.
+	**/
+	public static function print( v : Dynamic ) : Void {
+		untyped __java__("java.lang.System.out.print(v)");
```

Comment:
ouch, does it really have to be `untyped __java__`? i'm pretty sure you can access this api properly in haxe, and then you can also inline those functions.